### PR TITLE
feat(stage/wercker): Implemented a Wercker Pipeline Stage for triggering Wercker Builds

### PR DESCRIPTION
**Pull request by desagar**
- Wercker CI as a pipeline Stage in Spinnaker

Related PRs (Wercker CI build trigger):
https://github.com/spinnaker/halyard/pull/960
https://github.com/spinnaker/igor/pull/297
https://github.com/spinnaker/deck/pull/5519
https://github.com/spinnaker/echo/pull/295
https://github.com/spinnaker/gate/pull/574

@markxnelson

### CODE REVIEW

File: orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import org.springframework.mock.env.MockEnvironment
+import retrofit.RetrofitError
+import retrofit.client.Response
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+
+class MonitorWerckerJobStartedTaskSpec extends Specification {
+  def environment = new MockEnvironment()
+  def buildArtifactFilter = new BuildArtifactFilter(environment: environment)
```
Comment (ezimanyi) [inline orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy]: This variable (+import) is unused

File: orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    and:
+    task.buildService = Stub(BuildService) {
+      getBuild(stage.context.queuedBuild, stage.context.master, stage.context.job) >> [result: result, number: buildNumber]
+    }
+
+    expect:
+    task.execute(stage).status == expectedExecutionStatus
+
+    where:
+    result | buildNumber | expectedExecutionStatus
```
Comment (ezimanyi) [inline orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy]: More readable to align the columns in a data table test

File: orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy (hunk 1 above)
Comment (shihchang) [reply orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy]: cleaned up

File: orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy (hunk 2 above)
Comment (shihchang) [reply orca-igor/src/test/groovy/com/netflix/spinnaker/orca/igor/tasks/MonitorWerckerJobStartedTaskSpec.groovy]: updated

**Pull request merged**